### ISSUE TITLE
Do not overwriting null (no exist) values

### ISSUE THREAD

**Issue by eikebartels** [tristanhimmelman/AlamofireObjectMapper#142]
The following scenario. (using Realm) 

I'm parsing an object i.e. 

```
{
 id=10
 name= "awesome"
 description = "awesome description"
}
```

at a later state I want to override the object with the following 

```
{
 id=10
 name= "awesome 2"
}
```

I would like to keep the `description` value and only override `name` 

What would be the solution to do this? 

https://github.com/Hearst-DD/ObjectMapper/issues/555

**Comment by A1iAshoor** [comment]
+1

**Comment by tristanhimmelman** [comment]
Closing this as this was resolved in Hearst-DD/ObjectMapper#555
